File: c#/BEE 1010.cs
using System;

namespace beecrowd
{
    class BEE1010
    {
        static void Main(string[] args)
        {
            string[] input1 = Console.ReadLine().Split();
            string[] input2 = Console.ReadLine().Split();

            int cod1 = int.Parse(input1[0]);
            int quant1 = int.Parse(input1[1]);
            double valor1 = double.Parse(input1[2]);

            int cod2 = int.Parse(input2[0]);
            int quant2 = int.Parse(input2[1]);
            double valor2 = double.Parse(input2[2]);

            double valorTotal = (quant1 * valor1) + (quant2 * valor2);

            Console.WriteLine($"VALOR A PAGAR: R$ {valorTotal.ToString("F2")}");
        }
    }
}

File: python/BEE 1010.py
cod1, quant1, valor1 = input().split()
cod2, quant2, valor2 = input().split()

cod1 = int(cod1)
quant1 = int(quant1)
valor1 = float(valor1)

cod2 = int(cod2)
quant2 = int(quant2)
valor2 = float(valor2)

valorTotal = (quant1 * valor1) + (quant2 * valor2)

print(f'VALOR A PAGAR: R$ {valorTotal:.{2}f}')